torch/quantization/quantize_script.py

```diff
@@ -35,13 +35,15 @@ def _prepare_script(model, qconfig_dict, inplace=False, quant_type=QuantType.STA
     if not all(isinstance(x, str) for x in qconfig_dict.keys()):
         raise ValueError('qconfig_dict should only contain names(str) as keys.')
     scripted_qconfig_dict = script_qconfig_dict(qconfig_dict)
-    model = wrap_cpp_module(torch._C._jit_pass_fold_convbn(model._c))
+    torch._C._jit_pass_dedup_module_uses(model._c)
+    model = fuse_conv_bn_script(model)
     return wrap_cpp_module(torch._C._jit_pass_insert_observers(model._c,
                                                                'forward',
                                                                scripted_qconfig_dict,
                                                                inplace,
                                                                quant_type))
-
+def fuse_conv_bn_script(model):
```

Comment:
can you move this to a place before this function is used? also please keep one empty line between function definitions

Reply:
could you define this function in https://github.com/pytorch/pytorch/blob/master/torch/nn/utils/fusion.py?